orchprovider/k8s/v1/k8s.go

```diff
@@ -877,6 +878,19 @@ func (k *k8sOrchestrator) createControllerDeployment(volProProfile volProfile.Vo
 		ctrlLabelSpec[string(v1.ApplicationSelectorKey)] = appLV
 	}
 
+	//The replication count will be specified as REPLICATION_FACTOR
+	//  ENV for jiva controller. The controller will use this value
+	//  to determine if quorum is available for making volume as read-write.
+	//  For example,
+	//   - if replication factor is 1, volume will be marked as RW when one replica connects
+	//   - if replication factor is 3, volume will be marked as RW when when atleast 2 replica connect
+	//  Similar logic will be applied to turn the volume into RO, when qorum is lost.
+	//Note: When kubectl scale up/down is done, this ENV needs to be patched. 
```

Comment:
When kubectl scale up/down is done to the Replica deployment, this ENV i.e. Controller deployment needs to be patched.